docs/root/configuration/http/http_filters/admission_control_filter.rst

```diff
@@ -24,7 +24,12 @@ The probability that the filter will reject a request is as follows:
 
 .. math::
 
-   P_{reject} = {(\frac{n_{total} - s}{n_{total} + 1})}^\frac{1}{aggression}
+  P_{reject} = \left\{
+  \begin{array}{cl}
+  0 & \ (rps < rps\_threshold) \\
+  min({(\frac{n_{total} - s}{n_{total} + 1})}^\frac{1}{aggression}\ ,\ max\_reject\_probability) & \ (rps \geq rps\_threshold)
+  \end{array}
+  \right.
```

Comment:
Can you include a screenshot of this rendered in the docs? 

I also don't think it's necessary to change the function here as you're introducing circumstances under which this function applies, not changes to the behavior of the function.

Reply:
[screenshot here](https://docs.google.com/document/d/1Axe_bMuFk6-dbJ6oxhoI9LcRtIeT-miXbN0R7fz0ES8/edit?usp=sharing)

Your comment reminds me that this does make the formula more complicated. It may be more appropriate to explain these two configuration items after the original formula.